force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -5864,6 +5864,62 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Amount</shortDescription>
         <value>Amount</value>
     </labels>
+    <labels>
+        <fullName>commonPaymentMethod</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Payment Method</shortDescription>
+        <value>Payment Method</value>
+    </labels>
+    <labels>
+        <fullName>dateStarted</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Date Started</shortDescription>
+        <value>Date Started</value>
+    </labels>
+    <labels>
+        <fullName>lastDonationDate</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Last Donation Date</shortDescription>
+        <value>Last Donation Date</value>
```

Comment:
Can we use the npe03__Last_Payment_Date__c field on the Recurring Donations object for this label instead? The label for that field matches up with what's in the custom label here, "Last Donation Date".

Reply:
done